### PR TITLE
File structure

### PR DESCRIPTION
**Pull request by SajaLahaleeh**

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const router = require("./router");
+const express = require("express");
+const path = require("path");
+
+const app = express();
+app.use(express.static(path.join(__dirname, "..", "client", "build")));
+app.use(express.json());
+app.use(express.urlencoded());
+app.use(router);
```
Comment (FarahZaqout) [inline server/app.js]: add the `api` prefix to this.

File: server/server.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const PORT = process.env.PORT || 4000;
```
Comment (FarahZaqout) [inline server/server.js]: I suggest naming this file to index (default entry point in nodejs)

**Comment by SajaLahaleeh** [comment]
related to #29 @FarahZaqout

File: client/src/App.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import './App.css';
+
+function App() {
+  return (
+   <>loading</>
```
Comment (FarahZaqout) [inline client/src/App.js]: react router should be added to the project set up.

File: client/README.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (jema28) [inline client/README.md]: Please delete this README.

File: server/server.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+const PORT = process.env.PORT || 4000
+const app = require('./app')
+
+app.listen(PORT, () => {
+  console.log(`localhost: ${PORT}`)
```
Comment (jema28) [inline server/server.js]: Please add link here e.g. `App running on http://localhost:${PORT}`

File: server/server.js (hunk 2 above)
Comment (SajaLahaleeh) [reply server/server.js]: I did it @FarahZaqout

**Pull request merged**